index.html

```diff
@@ -16,7 +16,7 @@
 	</head>
 	<body>
 		<nav class="prompt">
-
+      <header class="prompt__header js-header"></header>
```

Comment:
I'm not sure why the formatting here looks weird on Github, because in vim it looks fine. I'm looking into it now.